.gitignore

```diff
@@ -52,11 +52,6 @@ coverage.xml
 # Sphinx documentation
 docs/_build/
 
-# PyBuilder
-target/
-!yellowbrick/target/
-!docs/api/target/
```

Comment:
This was causing some weirdness with my git -- I don't know anyone who uses PyBuilder so I just nixed it from the .gitignore and the issues were resolved.

Reply:
Interesting - looks like this line has been in .gitignore since the beginning, just added as part of a generic list, so I'm totally good with cutting it. Out of curiosity, what IDE/editor were you using when the odd git stuff was happening? Just asking because I'm constantly switching between my home machine set-up (atom.io) and my work machine (pycharm), and always have trouble keeping track. I think it's usually IntelliJ weirdness for me though.

Reply:
I'm using Atom -- though seriously considering switching over to VS code. 

PyBuilder is a continuous build tool that is mostly directed towards applications; so I don't think we have to worry about it for library development. And yes, it was part of the default GitHub python .gitignore file that we just added by default.